bool is_sim
int16 robot_num
robotMotionGroup[] robotMotionGroup_list
---
bool is_success
string info
